back-end/developerverse/models.py

```diff
@@ -1,3 +1,9 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
 
 # Create your models here.
+class User(AbstractUser):
+    email = models.EmailField(unique=True)
+
+    USERNAME_FIELD = 'email'
+    REQUIRED_FIELDS = []
```

Comment:
Looks like there's some special fields being defined here. Can you leave a comment in the source code clarifying what these properties are for?